### PR TITLE
Updated sample yaml for mesos policy.

### PR DESCRIPTION
**Pull request by djosborne**

### CODE REVIEW

File: docs/mesos/UsageGuideUnifiedCNI.md [hunk 1]
```diff
@@ -103,22 +103,26 @@ The above Marathon Application Definition has assigned the labels `app=frontend`
...
-    action: allow
-  outbound_rules:
-  - protocol: tcp
-    dst_selector: app == 'database'
-    dst_ports: [6379]
-    action: allow
+  ingress:
+  - action: allow
+    protocol: tcp
+    source:
```
Comment (caseydavenport) [inline docs/mesos/UsageGuideUnifiedCNI.md]: Shouldn't this be destination?

File: docs/mesos/UsageGuideUnifiedCNI.md [hunk 2]
```diff
@@ -103,22 +103,26 @@ The above Marathon Application Definition has assigned the labels `app=frontend`
...
+      ports:
+      - 80
+  egress:
+  - action: allow
+    protocol: tcp
+    selector: app == 'database'
+    source:
```
Comment (caseydavenport) [inline docs/mesos/UsageGuideUnifiedCNI.md]: Same here

**Comment by djosborne** [comment]
Fixes #1134

File: docs/mesos/UsageGuideUnifiedCNI.md [hunk 3]
```diff
@@ -103,29 +103,33 @@ The above Marathon Application Definition has assigned the labels `app=frontend`
...
+  - action: allow
+    protocol: tcp
+    selector: app == 'database'
+    destination:
+      ports:
+      - 6379
 ```
 
-[The new calicoctl](https://github.com/tigera/libcalico-go) can be used to register this policy spec:
+[The new calicoctl](https://github.com/tigera/libcalico-go/releases/tag/v1.0.0-alpha.1) can be used to register this policy spec:
```
Comment (caseydavenport) [inline docs/mesos/UsageGuideUnifiedCNI.md]: I'd call this "this alpha version" rather than "the new"

**Comment by caseydavenport** [comment]
One comment, then LGTM

**Pull request merged**